src/tiled/tilesetdocument.h

```diff
@@ -22,13 +22,12 @@
 
 #include "document.h"
 #include "tileset.h"
+#include "tilesetformat.h"
```

Comment:
I believe you shouldn't need this include and can keep relying on the forward declaration.

Reply:
Same reasoning as described above for ``mapdocument.h``